api/server/object.go

```diff
@@ -88,18 +86,14 @@ func (tapi ToDDApi) DeleteObject(w http.ResponseWriter, r *http.Request) {
 
 	deleteInfo := make(map[string]string)
 
-	body, err := ioutil.ReadAll(r.Body)
+	err := json.NewDecoder(r.Body).Decode(&deleteInfo)
```

Comment:
NewDecoder doesn't return an error (`func NewEncoder(w io.Writer) *Encoder`), it's just a factory function for an Encoder struct. This wouldn't be possible if NewDecoder had multiple returns because Go doesn't allow chaining off of a function that returns multiple values.